src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,211 @@
+package Duke;
+import Duke.todoList.task.listTypes;
+import Duke.todoList.todoList;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
+public class Duke {
+
+    public static void dukeStartup(){
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+
+        System.out.println("____________________________________________________________\n" +
+                " Hello! I'm Duke\n" +
+                " What can I do for you?\n" +
+                "____________________________________________________________\n" );
+
+    }
+
+    public static void dukeResponse(String output){
+        System.out.println("____________________________________________________________\n" +
+                output + "\n" +
+                "____________________________________________________________\n");
+
+    }
+
+    public static String command_parser(todoList list, String command){
+        String[] tokens = command.split(" ");
+        int itemStartIndex = command.indexOf(" ") + 1;
+        int itemEndIndex;
+        int dateStartIndex;
+        String out = "";
+        String desc = "";
+        String date = "";
+        switch(tokens[0]){
+        case "list": // If command is list, print all tasks in list
+            out = list.listItems();
+            break;
+        case "deadline": // If command is deadline, parse the task and by date, print confirmation
+            itemEndIndex = command.indexOf("/by") - 1;
+            dateStartIndex = itemEndIndex + 5;
+            try {
+                desc = command.substring(itemStartIndex, itemEndIndex);
+                date = command.substring(dateStartIndex);
+            }catch(StringIndexOutOfBoundsException e) { // if task or by date was not entered, catch error
```

Comment:
Consider leaving a space after `}`, like in these conventions [here](https://se-education.org/guides/conventions/java/basic.html#layout). You have a couple of similar instances below that could be revised.